docker/scripts/clean.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+# Run this shell script to rm existing CL Docker images and pull the latest, freshest ones.
+# Use when:
+## 1) Merging `main` into a working branch;
+## 2) Forking off of `main` to start a new working branch
+# Run it before using the build script. Usage of both of these shell scripts is optional.
+
+# Mostly from here: https://stackoverflow.com/a/39127792/3256077
+
+# stop all running CL containers
+docker-compose -f ../courtlistener/docker-compose.yml stop
+# rm said containers
+docker-compose -f ../courtlistener/docker-compose.yml rm --force
+# rm the locally-stored CL images themselves
+docker rmi $(docker-compose -f ../courtlistener/docker-compose.yml config | awk '/freelawproject/{if ($1 == "image:") print $2;}')
+# pull the latest images
+docker-compose -f ../courtlistener/docker-compose.yml pull
```

Comment:
I'd take this off of a cleaning script. When you start docker, it'll pull automatically, and it feels weird for a cleaning script to bring in more stuff you may or may not want.

Reply:
I think you're right that it does not belong in the clean script.

However, I am not sure that running `docker-compose up` actually pulls the latest images by default. I believe that this is only the case if the `pull_policy` config variable is set for a service: https://docs.docker.com/compose/compose-file/05-services/#pull_policy 

It may make sense to set that for certain services. IMO, we should use `build` instead of `image` for `cl-django` and point it at the Dockerfile. Ultimately, then, we can just remove the build script that is included in this PR (as well as remove this line from the clean script).

Reply:
I'm happy to try that, sure.